test/parallel/test-zlib-zero-windowBits.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const zlib = require('zlib');
+
+
+// windowBits is a special case in zlib. On the compression side, 0 is invalid.
+// On the decompression side, it indicates that zlib should use the value from
+// the header of the compressed stream.
+{
+  const inflate = zlib.createInflate({windowBits: 0});
+  assert(inflate instanceof zlib.Inflate);
+}
+
+{
+  const inflateRaw = zlib.createInflateRaw({windowBits: 0});
+  assert(inflateRaw instanceof zlib.InflateRaw);
+}
+
+{
+  const gunzip = zlib.createGunzip({windowBits: 0});
+  assert(gunzip instanceof zlib.Gunzip);
+}
+
+{
+  const unzip = zlib.createUnzip({windowBits: 0});
+  assert(unzip instanceof zlib.Unzip);
+}
+
+{
+  let failed = false;
+  let deflate = null;
+  try {
+    deflate = zlib.createDeflate({windowBits: 0});
```

Comment:
This is already tested in `test-zlib-deflate-constructors.js` so this is probably redundant.